### PR TITLE
added some api docs

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: apps/apiv1/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""Restful api for the packaginator, based on `django-piston`_.
```
Comment (ptone) [inline apps/apiv1/__init__.py:1]: actually its based on tastypie, not piston

File: docs/reference.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Development reference documentation
+===================================
+
+.. currentmodule:: apps
+
+Top level module ``apps``
+=========================
+.. automodule:: apps
+
+All of the modules listed below are the sub-applications.
```
Comment (ptone) [inline docs/reference.rst:11]: perhaps change "the sub-applications" to "Packiginator's core applications"

File: docs/conf.py [hunk 3]
```diff
@@ -13,10 +13,20 @@
 
 import sys, os
 
+# todo: see if VIRTUAL_ENV variable is in the paths
+# and if it is not, add to os.path - this is necessary when sphinx is 
+# installed only in the sitewide packages directory
```
Comment (ptone) [inline docs/conf.py:18]: Doing this, I was able to get sphinx to build using the project virtualenv:

```
make html SPHINXBUILD='python $(shell which sphinx-build)'
```

**Comment by ptone** [comment]
The new docs render beautifully - curious to see how RTD handles this kind of setup

**Comment by pydanny** [comment]
http://readthedocs.org/docs/packaginator/en/latest/reference.html

Took some wrestling to get it to work in RTD, but OMG this looks awesome!

**Pull request merged**